server/surveyRdb/dbActions/tableViewQuery.js

```diff
@@ -68,12 +69,12 @@ const queryRootTableByRecordKeys = async (survey, recordUuid, client) => {
     keyNodes.map(
       async node => {
         const colValue = await getColValue(node)
-        return `${getColName(node)} ${colValue === null ? ' IS NULL' : " = '" + colValue + "'"}`
+        return `${getColName(node)} ${colValue === null ? ' IS NULL' : ' = \'' + colValue + '\''}`
```

Comment:
is this escaping needed ?

Reply:
It's not... Must have been the editor...